executor/adapter.go

```diff
@@ -876,6 +876,9 @@ func (a *ExecStmt) SummaryStmt() {
 	}
 	sessVars.SetPrevStmtDigest(digest)
 
+	plan := plannercore.EncodePlan(a.Plan)
```

Comment:
No  performance impact?

Reply:
But it look ok from the result of benchmark Report...

Reply:
I changed it to recording the plan only at the first time.